src/modules/I18n/i18next-scanner.config.js

```diff
@@ -4,13 +4,25 @@ const {
 const fg = require('fast-glob');
 const typescriptTransform = require('i18next-scanner-typescript');
 
-const namespaces = fg
+const moduleNamespaces = fg
   .sync(['src/modules/*', 'src/modulesAdmin/*', 'src/api/modules/*'], {
```

Comment:
you can remove `src/modulesAdmin/*` as it does not exist

Reply:
Done, thx.